test/test_document.rb

```diff
@@ -44,6 +44,16 @@ def assert_equal_value(key, one, other)
       assert_equal "foo.bar", @document.data["whatever"]
     end
 
+    should "be able to jsonify its data" do
+      require 'json'
```

Comment:
* Do not require within a test.  This is bad practice, if the library is not available you're doing it wrong.